packages/gatsby-theme/src/components/MobileMenu/MobileMenu.stories.mdx

```diff
@@ -53,23 +53,30 @@ export const BComponent = (args) => {
   )
 }
 
-<Meta title='components/MobileMenu' component={MobileMenu} />
+<Meta
+  title='components/MobileMenu'
+  component={MobileMenu}
+  subcomponents={{
+    MobileMenuItem: MobileMenu.Item,
+    MobileMenuNavbar: MobileMenu.Navbar,
+  }}
+/>
 
 # MobileMenu
 
-Use this MobileMenu
+Use this component to insert a MobileMenu.
 
-<Props of={MobileMenu} />
+<ArgsTable of={MobileMenu} />
 
 ### Default
 
-<Preview>
+<Canvas>
   <Story name='Default'>{BComponent.bind({})}</Story>
-</Preview>
+</Canvas>
```

Comment:
For some reason, the Docs tab is not working on the storybook. The overlay covers the entire screen. We should check to see if there is an easy fix, otherwise we can take a look into it later.

![image](https://user-images.githubusercontent.com/41605725/108225914-01dbfe00-7102-11eb-94d3-79ac7c4b8dd9.png)

Reply:
Thank you @dinartecc I removed the megamenu story because it causes conflicts with the docs from the storybook